odl/solvers/advanced/forward_backward.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Implementation of the Forward-Backward splitting algorithm """
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+from odl.operator import Operator
+
+
+__all__ = ('forward_backward_pd',)
+
+
+def forward_backward_pd(x, prox_f, prox_cc_g, L, grad_h, tau, sigma, niter,
```

Comment:
Depends. The "pd" suffix indicates that it's the primal-dual one and not the "vanilla" version without operator. Since this is somewhat the forefront of algorithm prototyping, I'd be less worried here to change names as needed later. Currently, this seems like an okay name to me.